backend/requirements.txt

```diff
@@ -1,3 +1,3 @@
 # Packages for production environment
-flask>=1.0.2
-flask-script>=2.0.6
\ No newline at end of file
+flask-script==2.0.6
```

Comment:
So I notice a warning on `flask-script` site: 

> Warning: While the maintainers are willing to merge PR’s, they are not actively developing features. As of Flask 0.11, Flask includes a built-in CLI tool, and that may fit your needs better.

Should we not use this and use the built in [Flask CLI](http://flask.pocoo.org/docs/1.0/cli/)? Seems like it has similar features to what you have coded up here.

Reply:
I think we’ll be okay just using the built in flask CL. What do you think, Adrian?

Reply:
hey this will break a lot of things, I am using flask-script for now, but porting it to flask CLI later. Atm, I just don't like how flask deals with CLI so I am using flask-script to test stuff out.

Reply:
Ah okay. No worries, just trying to understand :smile: